.travis.yml

```diff
@@ -2,10 +2,10 @@ os: linux
 dist: focal
 language: node_js
 node_js:
-  - "10.2.0"
+  - "12.22.0"
```

Comment:
[Node v12.22.0][1] was the first version to support ES modules without requiring a command line flag.


[1]: https://nodejs.org/en/blog/release/v12.22.0/